css/cssom/FontFaceRulevsLocal-ref.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>FontFaceArial vs Local Arial</title>
+<link rel="author" title="Adam Argyle" href="mailto:argyle@google.com">
+<link rel="help" href="https://drafts.csswg.org/css-fonts-4/">
+<style>
+  @font-face {
+    font-family: "Local Arial";
+    src: local(Arial);
+  }
+
+  h1 {
+    font-family: Arial; 
+  }
+
+  h1 {
+    font-family: Arial;
+  }
+</style>
+<h1><p>Tests pass if these render identical</p></h1>
```

Comment:
Since the screenshots of the test and ref are compared, you should be able to use just `font-family: Arial` in the ref, and then just the `@font-face` stuff in the test. That's at least the minimum test that would exercise the bug and fail.

To make it more obvious what the problem is, though, you could include both forms in the test, but just `font-family: Arial` in the ref. That way one can see that there's a difference by looking at the test. I suspect some reviewers might push back against as not minimal enough, but I think it would be OK :)